pkg/oci/ccm.go

```diff
@@ -68,17 +71,21 @@ var _ cloudprovider.Interface = &CloudProvider{}
 
 // NewCloudProvider creates a new oci.CloudProvider.
 func NewCloudProvider(config *Config) (cloudprovider.Interface, error) {
-	cp, err := buildConfigurationProvider(config)
+	logger := logutil.Logger()
+	defer logger.Sync()
+	zap.ReplaceGlobals(logger)
```

Comment:
Replacing it here means there will be logs emitted not following this logger since the CCM itself logs plus other various global things.

Reply:
Would you suggest swapping out the logger in `func main()` then?

Reply:
Yes